exist-versioning-release.md

```diff
@@ -93,40 +93,121 @@ That version number is also copied into the `META-INF/MANIFEST.MF` file of any J
 
 ## Release Process
 
-This section details concrete steps for creating and publishing product releases. Each section here assumes you are starting with a clean Git checkout of the `develop` branch from [https://github.com/eXist-db/exist.git](https://github.com/eXist-db/exist.git).
```

Comment:
As discussed in the calls, I believe it is important to always start a release from a clean `git checkout` of the `develop` branch. So I think this section should not be removed

Reply:
Yes, this is why we decided to make this a dedicated step. That way there is less chance to miss it.

Reply:
Thanks, I see that below now, but I do think it is better at the beginning or at least also a note at the beginning.